src/material-experimental/mdc-form-field/_mdc-text-field-structure-overrides.scss

```diff
@@ -74,6 +74,12 @@
   // Note: Increased specificity needed here since MDC overwrites the padding on `:focus`.
   .mat-mdc-form-field .mat-mdc-text-field-wrapper.mdc-text-field .mdc-notched-outline__notch {
     padding-top: 0;
+    // MDC adds a 8px padding on the right/left (depending on LTR or RTL mode) that causes the
```

Comment:
Is this something that should be fixed in the MDC code or is it related to some custom treatment we have for the label?